### PR TITLE
Feature/prime 1269 escalation

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-dotnet-webapi/Models/Notes/EnrolleeNote.cs [hunk 1]
```diff
@@ -9,9 +9,13 @@ public class EnrolleeNote : BaseAdjudicatorNote, IBaseEnrolleeNote
         public int EnrolleeId { get; set; }
 
         [JsonIgnore]
+        [NotMapped]
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/Notes/EnrolleeNote.cs:12]: Is this intentional?

**Review by AnaisHebert — APPROVED** [review]
Seems to work well! 🎈🎈🎈

File: prime-dotnet-webapi/Controllers/EnrolleesController.cs [hunk 2]
```diff
@@ -63,7 +63,10 @@ public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions s
         {
             if (User.HasAdminView())
             {
-                return Ok(ApiResponse.Result(await _enrolleeService.GetEnrolleesAsync(searchOptions)));
+                var notifiedIds = await _enrolleeService.GetNotifiedEnrolleeIdsForAdminAsync(User);
+                var enrollees = await _enrolleeService.GetEnrolleesAsync(searchOptions);
+                var result = enrollees.Select(e => notifiedIds.Contains(e.Id) ? e.SetNotification(true) : e.SetNotification(false));
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/EnrolleesController.cs:68]: Here you could just do
`foreach(var enrollee in enrollees)
{
    e.HasNotification = notifiedIds.Contains(enrollee.Id)
}
`

**Review by mtpultz — APPROVED** [review]

File: prime-dotnet-webapi/Models/SiteRegistration/Site.cs [hunk 3]
```diff
@@ -3,6 +3,7 @@
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
+using DelegateDecompiler;
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/SiteRegistration/Site.cs:6]: I assume you dont need this

**Review by james-hollinger — APPROVED** [review]

**Pull request merged**